BuildResidentialHPXML/measure.rb

```diff
@@ -5080,10 +5087,36 @@ def self.set_ducts(args, hvac_distribution)
       ducts_supply_surface_area = args[:ducts_supply_surface_area].get
     end
 
+    if args[:ducts_supply_surface_area_fraction].is_initialized
+      ducts_supply_area_fraction = args[:ducts_supply_surface_area_fraction].get
+    end
+
     if args[:ducts_return_surface_area].is_initialized
       ducts_return_surface_area = args[:ducts_return_surface_area].get
     end
 
+    if args[:ducts_return_surface_area_fraction].is_initialized
+      ducts_return_area_fraction = args[:ducts_return_surface_area_fraction].get
+    end
+
+    if (not ducts_supply_location.nil?) && ducts_supply_surface_area.nil? && ducts_supply_area_fraction.nil?
```

Comment:
Should there be a check to fail if both surface_area and area_fraction are specified? Or does it simply take precedence over another?

Reply:
The specified duct area takes precedence, since this is what is needed for simulation. (When fractions are provided, we estimate duct areas from them.) I've gone ahead and clarified this in the documentation.